### PR TITLE
Add tests for RequestHelpers.GetItemTypeStrings

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

**Review by crobibero — APPROVED** [review]

File: tests/Jellyfin.Api.Tests/Jellyfin.Api.Tests.csproj [hunk 1]
```diff
@@ -41,4 +41,10 @@
     <CodeAnalysisRuleSet>../jellyfin-tests.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>Jellyfin.Api.Tests</_Parameter1>
+    </AssemblyAttribute>
+  </ItemGroup>
+
```
Comment (Ullmie02) [inline tests/Jellyfin.Api.Tests/Jellyfin.Api.Tests.csproj:49]:
```suggestion
```
Don't think this is needed

Comment (Bond-009) [reply tests/Jellyfin.Api.Tests/Jellyfin.Api.Tests.csproj:49]: We do, the function is internal

Comment (Ullmie02) [reply tests/Jellyfin.Api.Tests/Jellyfin.Api.Tests.csproj:49]: Yeah I know but expose the internals from `Jellyfin.Api.Tests` to itself doesn't make any sense

Comment (Bond-009) [reply tests/Jellyfin.Api.Tests/Jellyfin.Api.Tests.csproj:49]: Oh wait...how did this happen

**Review by Ullmie02 — APPROVED** [review]

**Pull request merged**